lib/core/db_models/old/job_data_model.dart

```diff
@@ -26,7 +27,8 @@ class JobDataModel {
     this.issuesArray,
     this.workersArray,
     this.locationsArray,
-    this.totalShiftsCount =0,
+    this.rate,
```

Comment:
if totalshift count is defaulted does rate need defaulting too?

Reply:
i am giving the default value for rate during the parsing